packages/vuetify/src/components/VInput/VInput.tsx

```diff
@@ -190,6 +190,6 @@ export const VInput = genericComponent<VInputSlots>()({
 export type VInput = InstanceType<typeof VInput>
 
 export function filterInputProps (props: Record<string, unknown>) {
```

Comment:
This function can be removed and replaced with `VInput.filterProps` if it isn't handling events differently anymore.

Reply:
nice ! 🎉